tests/acceptance/features/webUISecureView/main.feature

```diff
@@ -0,0 +1,284 @@
+@webUI @insulated @disablePreviews
+Feature: Secure View
+  As an admin
+  I want to be able to enable secure view
+  So that users can share files with very restricted access to documents
+
+  Scenario: Admin enables the secure view through webUI
+    Given the administrator has browsed to the admin additional settings page
+    When the administrator enables secure view using the webUI
+    Then the config key "secure_view_option" of app "richdocuments" should have value "true"
+
+  Scenario: Admin enables the secure view, disables print permissions through webUI
+    Given the administrator has browsed to the admin additional settings page
+    When the administrator enables secure view using the webUI
+    And the administrator disables print permission in secure view using the webUI
+    Then the config key "secure_view_option" of app "richdocuments" should have value "true"
+    And the config key "secure_view_can_print_default" of app "richdocuments" should have value "false"
+
+  Scenario: Admin enables the secure view, disables print permissions through webUI
+    Given the administrator has browsed to the admin additional settings page
+    When the administrator enables secure view using the webUI
+    And the administrator disables watermark permission in secure view using the webUI
+    Then the config key "secure_view_option" of app "richdocuments" should have value "true"
+    And the config key "secure_view_has_watermark_default" of app "richdocuments" should have value "false"
+
+  Scenario: Admin enables the secure view, disables print and watermark permissions through webUI
+    Given the administrator has browsed to the admin additional settings page
+    When the administrator enables secure view using the webUI
+    And the administrator disables watermark permission in secure view using the webUI
+    And the administrator disables print permission in secure view using the webUI
+    Then the config key "secure_view_option" of app "richdocuments" should have value "true"
+    And the config key "secure_view_has_watermark_default" of app "richdocuments" should have value "false"
+    And the config key "secure_view_can_print_default" of app "richdocuments" should have value "false"
+
+  Scenario: Admin enables the secure view using occ command, disables print and enables it again through webUI
+    Given the administrator has added config key "secure_view_option" with value "true" in app "richdocuments"
```

Comment:
Is it correct to use the string `"true"` here, or should be rather use a new step to set the type also, like `theAdministratorAddsSystemConfigKeyWithValueUsingTheOccCommand()` ?

Reply:
The webui sets it to string `"true"`, so, it should be fine here.